recipes/plf_list/all/conanfile.py

```diff
@@ -27,7 +28,13 @@ def source(self):
             destination=self.source_folder, strip_root=True)
 
     def build(self):
-        pass
+        # apple-clang 14 defaults C++ 20, but it doesn't support C++20 features little
+        # https://www.reddit.com/r/cpp/comments/v6ydea/xcode_now_defaults_to_c20/
+        if Version(self.version) >= "2.50":
+            replace_in_file(self, os.path.join(self.source_folder, "plf_list.h"),
+            "if __cplusplus > 201704L && ((defined(_LIBCPP_VERSION) && _LIBCPP_VERSION >= 13) || !defined(_LIBCPP_VERSION)) && ((defined(__clang__) && (__clang_major__ >= 13)) || (defined(__GNUC__) && __GNUC__ >= 10) || (!defined(__clang__) && !defined(__GNUC__)))",
+            "if __cplusplus > 201704L && ((defined(_LIBCPP_VERSION) && _LIBCPP_VERSION >= 13) || !defined(_LIBCPP_VERSION)) && (!defined(__APPLE_CC__) && (defined(__clang__) && (__clang_major__ >= 13)) || (defined(__GNUC__) && __GNUC__ >= 10) || (!defined(__clang__) && !defined(__GNUC__)))"
+            )
```

Comment:
no_copy_source must be removed if there is a patch

Reply:
@SpaceIm 
I missed it.
Thank you.

Reply:
Is it possible to move this to a patch file please?

Reply:
@SpaceIm 
I agree with you.
Let me try.

Reply:
Fixed